cmd/jujud/agent/bootstrap_test.go

```diff
@@ -123,7 +123,16 @@ func (s *BootstrapSuite) SetUpTest(c *gc.C) {
 	s.makeTestModel(c)
 
 	// Create fake tools.tar.gz and downloaded-tools.txt.
+	//
+	// CurrentVersion() currently hardcodes Release to "ubuntu" to match
+	// the expectations of other test suites wrt. the underlying host where
+	// the tests are running. However, this test is an exception as we
+	// are testing the bootstrap flow which uses the actual OS type to
+	// select agent binaries. Therefore, we need to specify the appropriate
+	// release value here.
```

Comment:
Bootstrap is only supported on ubuntu so we should test with agent metadata set to ubuntu rather than the host os on which the test is run